redact/v4/data_models.py

```diff
@@ -49,6 +52,72 @@ class JobArguments(BaseModel):
     lp_determination_threshold: Optional[float] = Field(None, ge=0, le=1)
     face_determination_threshold: Optional[float] = Field(None, ge=0, le=1)
     status_webhook_url: Optional[AnyHttpUrl] = None
+    areas_of_interest: Optional[str] = None
```

Comment:
Shouldn't it be a list of list's or a list of tuples? Why is it a string?

Reply:
we do not need to do anything with except for validation. and it will be converted into a string for a request later anyways. why not keep it as a string?